src/server_manager/messages/master_messages.json

```diff
@@ -1244,8 +1244,13 @@
     }
   },
   "share_invite": {
```

Comment:
I noticed that it's hard to tell which messages are text and which are html. I wish we named all HTML messages with a `_html` prefix, like `share_invite_html`.